src/pb_stub.cc

```diff
@@ -556,6 +537,7 @@ Stub::Initialize(bi::managed_external_buffer::handle_t map_handle)
   c_python_backend_utils.attr("shared_memory") = py::cast(shm_pool_.get());
 
   async_event_loop_ = py::none();
+  background_futures_ = py::module_::import("builtins").attr("set")();
```

Comment:
```suggestion
  background_futures_ = py::set();
```

https://github.com/pybind/pybind11/blob/9b3a2000653a002abe0fe96ce8bd002f9a67f3d4/tests/test_pytypes.cpp#L160-L163

Reply:
Nice finding! Something I was looking for but couldn't find on the official docs. Updated: [Use pybind11 provided python set](https://github.com/triton-inference-server/python_backend/pull/365/commits/063fdfe6d8f040cdd1ea6a722bee123f5236ef97)